src/utils/format.js

```diff
@@ -130,7 +130,11 @@ export function formatFiatValue(value: ?BigNumber | number, currency?: string, o
   const formattedValue = formatValue(value, { decimalPlaces: 2, ...options });
   if (!formattedValue) return null;
 
-  return currency ? t('fiatValue', { value: formattedValue, symbol: getCurrencySymbol(currency) }) : formattedValue;
+  if (!currency) return formattedValue;
+
+  const currencyValue = t('fiatValue', { value: formattedValue, symbol: getCurrencySymbol(currency) });
+  // eslint-disable-next-line max-len
+  return BigNumber(formattedValue).lt(0.01) && BigNumber(formattedValue).gt(0) ? ('< ').concat(currencyValue) : currencyValue;
```

Comment:
Split this into multiple lines please to make it cleaner, for example:

```suggestion
  return BigNumber(formattedValue)
    .lt(0.01) && BigNumber(formattedValue).gt(0) ? ('< ').concat(currencyValue) : currencyValue;
```